roles/tendrl-httpd-ssl/tasks/main.yml

```diff
@@ -0,0 +1,62 @@
+---
+
+- debug:
+    var: httpd_server_name
+
+- debug:
+    var: httpd_ip_address
```

Comment:
Would it make sense, to use some defaults for `httpd_server_name` and for `httpd_ip_address`?

Reply:
Actually, yes.